### PR TITLE
Improve notification icon and navbar  (#25111)

### PR DESCRIPTION
**Pull request by GiteaBot**
Backport #25111 by @silverwind

Improvements to the notification icon and `<nav>`:

- Add a opaque color for header hover and use it, allowing the border to be the right color on hover (sadly, not otherwise possible with CSS, not even `color-mix`).
- Increase font size by 1px
- Use flexbox for slightly better text centering
- Reduce padding of user and add repo button, add margin on right side of user menu
- Remove the `following bar` wrapper on navbar

Screenshot 2023-06-07 at 00 07 08
Screenshot 2023-06-07 at 00 07 14
Screenshot 2023-06-07 at 00 07 36
Screenshot 2023-06-07 at 00 25 10
Screenshot 2023-06-07 at 00 35 45
Screenshot 2023-06-07 at 16 57 40

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
I think there is something wrong with branch protection 🤔

**Review by wxiaoguang — CHANGES_REQUESTED** [review]
don't backport.

it seems that it breaks mobile UI

image

**Comment by silverwind** [comment]
Mobile fix is included in https://github.com/go-gitea/gitea/pull/25134, just merge this and than the other backport.

File: templates/base/head.tmpl [hunk 1]
```diff
@@ -73,9 +73,7 @@
 		{{template "custom/body_inner_pre" .}}
 
 		{{if not .PageIsInstall}}
-			<div class="ui top secondary stackable main menu following bar light">
-				{{template "base/head_navbar" .}}
-			</div><!-- end bar -->
+			{{template "base/head_navbar" .}}
```
Comment (silverwind) [inline templates/base/head.tmpl:76]:
```suggestion
			{{template "base/head_navbar" .}} 
```

File: templates/base/head.tmpl [hunk 2]
```diff
@@ -73,9 +73,7 @@
...
```
Comment (silverwind) [inline templates/base/head.tmpl]:
```suggestion
			{{template "base/head_navbar" .}}
```

**Comment by silverwind** [comment]
Pushed some edits to get drone running, but still seems stuck

**Comment by silverwind** [comment]
Drone still unresponsive. Need a manual admin merge here.

**Comment by lunny** [comment]
> Pushed some edits to get drone running, but still seems stuck. PR will need a manual merge I assume.

> Drone still unresponsive. Need a manual admin merge here.

It's a wrong status check configuration. I have adjusted it.

**Comment by silverwind** [comment]
Going again, thanks. https://github.com/go-gitea/gitea/pull/25134 needs some reviews for the followup with the mobile fix this this.

**Pull request merged**